### PR TITLE
Pull request to add contact form in Hamayoun Portfolio 

### PR DESCRIPTION
**Pull request by Hamayounali**
# To Add contact form in Hamayoun portfolio 
1. Updated style.css file 
2. Updated index.html file

### CODE REVIEW

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# INVALID ❌ ❌ ❌ 

Hi @, 👋

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ❌
 - You have submitted a Pull Request that has not passed the linters test. Please check your linters before submitting the PR again.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

✅ Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_⚠️ Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

----
_Invalid Code Review Request does not count into the code reviews limit._

**Review by Kandy-Peter — CHANGES_REQUESTED** [review]
Hi @Hamayounali  

Good job so far! I know you have worked hard on making this milestone. You still need to work on an issue to go to the next project, but you are almost there!

almost-there-almost


## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- You  are following GitHub actions :heavy_check_mark: 
- Good commit history :heavy_check_mark: 

## Required Changes ♻️

- [x] While your design looks nice, it seems like you forgot to generate your hasd_id from [Formspree](https://formspree.io/) and this blocks users from submitting any message. Kindly, go to [Formspree](https://formspree.io/) and generate your unique `hash_id`. You can watch this short [video](https://youtu.be/H_MUJ--iAcc) for guidance. Your form action should look like this: 
```
   <form
      action="https://formspree.io/f/xwkyerln"
      method="POST"
   >
```

- [x] It would be better if you align your `submit` button to the center to stick the Figma design as required.
**_Your design_**

image

**The Figma design**
image

- [x] Kindly check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Hamayounali** [comment]
Hi Kandy Peter. Thank you for the nice suggestions, I have updated accordingly.

File: assets/css/style.css [hunk 1]
```diff
@@ -189,7 +189,7 @@
 }
 
 .tags-2 {
-  padding: 0px;
+  padding: 1px;
```
Comment (rustam-tolipov) [inline assets/css/style.css:192]: - [x] [OPTIONAL] GOOD JOB 👏🏼💥🎉✌🏼. I have a suggestion for you to improve your design. As you may know there is a space around your website as you can see down below. You can fix it by adding this code to body in css `margin: 0`. Good luck 🍀🙌🏼👍🏼

image

**Review by rustam-tolipov — APPROVED** [review]
## Hi @Hamayounali 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

Developer-512px-10

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: assets/css/style.css (hunk 1 above)
Comment (Hamayounali) [reply assets/css/style.css:192]: Thanks. Noted, but linter doesn't allow 0 value in CSS.

**Pull request merged**